lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,50 @@
+namespace :publishing_api do
+  desc "Publish special routes such as the homepage"
+  task :publish_special_routes => :environment do
+    require 'gds_api/publishing_api/special_route_publisher'
+
+    publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new
+
+    routes = [
+      {
+        content_id: "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a",
+        base_path: "/",
+        title: "GOV.UK homepage",
+      },
+      {
+        content_id: "ffcd9054-ee77-4539-978d-171a60eb4b2a",
```

Comment:
Is this meant to have the same base_path as the previous one?  If so, this looks quite odd, and I assume there's some "magic" happening here.  A comment to explain what path is being redirected to / here would be helpful.